.travis.yml

```diff
@@ -6,9 +6,19 @@ python:
     - "3.5"
     - "3.6"
 
+env:
+    - DB_URL=sqlite:///test.db
```

Comment:
@tomchristie I've left this is on so it achieves 100% test coverage as there is an exception to allow creation of sqlite engines: https://github.com/tomchristie/apistar/pull/48/files#diff-2c8f3f55d75d3cfd5148140210740646R25